src/app/components/homepage/NavBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {AppBar, Link, Toolbar, Typography} from '@material-ui/core';
+import { useTranslation } from 'react-i18next'
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+    fontWeight: 700
+  },
+}));
+
+export default function NavBar(props) {
+  const classes = useStyles();
+  const [t] = useTranslation();
+  return (
+    <div className={classes.root}>
+      <AppBar position="fixed" color="default">
+        <Toolbar>
+          <Link color="inherit" href="/" className={classes.title}>
+            <Typography variant="h6" color="default">
+              {t('self.title')}
+            </Typography>
+          </Link>
```

Comment:
The link is taking up almost the entire header.

![image](https://user-images.githubusercontent.com/30875400/111659157-c1989a00-87eb-11eb-8fbd-f606d4ce04dc.png)